.gitlab-ci.yml

```diff
@@ -41,13 +41,13 @@ variables:
   VCS_URL:
     description: |
       Repository or clone URL of project to scan.
-    value: 'https://github.com/jshttp/mime-types'
+    value: ''
```

Comment:
Did you consider also removing the `vcs.type` default value?

Reply:
I left VCS type defaults unchanged as I remembered without it ORT will throw a unclear error - and any repository in GitLab is always a Git repo.